packages/stark-core/package.json

```diff
@@ -1,6 +1,10 @@
 {
   "name": "@nationalbankbelgium/stark-core",
   "version": "0.0.0-PLACEHOLDER-VERSION",
+  "module": "./esm5/stark-core.js",
+  "es2015": "./es2015/stark-core.js",
+  "main": "bundles/stark-core.umd.js",
```

Comment:
Why is it no "main": "./bundles/stark-core.umd.js" as it is for "module" and "es2015" ?

Reply:
@SuperITMan 
I have implemened by following the documentation : 
https://docs.google.com/document/d/1CZC2rcpxffTDfRDs6p1cfbmKNLA6x5O-NtkJglDaBVs/preview

Kr,
Cédric